index.html

```diff
@@ -45,7 +45,7 @@ <h2 data-i18n="app.title"></h2>
                             <input class='col8' id='address' placeholder='' data-i18n="[placeholder]step1.addr"/>
                             <button class='col4' id='findme' data-i18n="step1.button"></button>
                             <br />
-                            Or <a id="use_my_location" href="#" onclick="return false">use my current location</a>.
+                            <a id="use_my_location" data-i18n="step1.use_location" href="#" onclick="return false">use my current location</a>.
```

Comment:
Yeah, I did this to make it easier to translate without having to embed the links in the translated string. I might have a look at bringing it back if we add more controls down there, like 'import from facebook' or similar.